run_composeEff_rooKeys.sh

```diff
@@ -22,6 +21,11 @@ ybin=${8}
 zbin=${9}
 ndiv=${10}
 totdiv=${11}
+year=${12}
+
+export SAMPLEDIR=/eos/cms/store/user/fiorendi/p5prime/effKDE/${year}/lmnr/
```

Comment:
Since a new user will need to customise this path, together with `HOME` and `CMSSWDIR`, I would suggest to keep the three of them together to avoid that one of them is forgotten. The line `year=${12}` would need to be moved as well at the beginning (and maybe also all the other variables' definition, to keep them together).

In addition, in line 42 of the [README](README.md) there is a pointer to the three lines to be customised, and it will need to be updated

Reply:
Otherwise, we could try to avoid the need for customisation, like passing `HOME` from the submit script (as argument, copying `$PWD`), setting `SAMPLEDIR=$HOME` (this is the default target of createDataset.cc, then if the user moves the datasets' files will need to customise this variable anyway), and either pointing `CMSSWDIR` to a directory in a public area, that any user can access, or passing it from the submit script

Reply:
solved with the easy option 1.